gateway/mockdecoder.js

```diff
@@ -0,0 +1,87 @@
+
+function decodeUplink(input) {
+    var decoded = milesightDeviceDecode(input.bytes);
+    return { data: decoded };
+}
+
+// Chirpstack v3
+function Decode(fPort, bytes) {
+    return DeviceDecode(bytes);
+}
+
+// The Things Network
+function Decoder(bytes, port) {
```

Comment:
why `Decoder` and `Decode`?

Reply:
Decoder is used by the things network and decode is used by chirpstack, I just modeled the mock decoder exactly how a real one looks like.

Reply:
(things network and chirpstack are both popular lorawan network servers)